pypeit/tests/test_metadata.py

```diff
@@ -1,15 +1,57 @@
 import os
 import glob
+import shutil
+
 import pytest
 
 import numpy as np
 
 from pypeit.par.util import parse_pypeit_file
 from pypeit.pypeitsetup import PypeItSetup
-from pypeit.tests.tstutils import dev_suite_required
+from pypeit.tests.tstutils import dev_suite_required, data_path
 from pypeit.metadata import PypeItMetaData
 from pypeit.spectrographs.util import load_spectrograph
-
+from pypeit.scripts import setup
+
+def test_read_combid():
```

Comment:
Nice!

Reply:
Thanks.  Ran into problems reading one of the pypeit files in the dev suite because it had these defined.  So I added a test to make sure I caught this failure mode.